### PR TITLE
Add functionality for displaying user location

### PR DESCRIPTION
**Pull request by dotslash21**
Fixes issue #7 

Signed-off-by: Arunangshu Biswas <arunangshubsws@gmail.com>

### CODE REVIEW

File: js/get_location.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  ipify_request.open("GET", "https://api.ipify.org", true);
+
+  ipify_request.onload = function() {
+    if (this.status >= 200 && this.status < 400) {
+      ip = this.response;
+
+      // Perform GET request to ip-api.com to get Client's Geo-Location
+      // based on IP address.
+      var ip_api_request = new XMLHttpRequest();
+      ip_api_request.open("GET", "http://ip-api.com/json/" + ip, true);
```
Comment (ayulockin) [inline js/get_location.js:21]: Nice choice of service provider.

**Review by ayulockin — COMMENTED** [review]
Screenshot (306)

Can you align the location logo with the text? 
They seem off. 

Great work :D @dotslash21 
You are killing it. Keep up the good work.

**Comment by dotslash21** [comment]
@ayulockin the Icon and the text should be aligned now.

**Comment by ayulockin** [comment]
They seem aligned. Nice work correcting it too. For now I am merging this pull request. 
👯‍♂

**Pull request merged**